doc/REST-interface.md

```diff
@@ -89,10 +89,12 @@ Only supports JSON as output format.
 * softforks : (array) status of softforks in progress
 
 #### Query UTXO set
-`GET /rest/getutxos/<checkmempool>/<txid>-<n>/<txid>-<n>/.../<txid>-<n>.<bin|hex|json>`
+`GET /rest/getutxos/<TXID>-<N>/<TXID>-<N>/.../<TXID>-<N>.<bin|hex|json>`
+`GET /rest/getutxos/checkmempool/<TXID>-<N>/<TXID>-<N>/.../<TXID>-<N>.<bin|hex|json>`
 
-The getutxo command allows querying of the UTXO set given a set of outpoints.
-See BIP64 for input and output serialisation:
+The getutxos endpoint allows querying the UTXO set given a set of outpoints.
+With the /checkmempool/ option the mempool is also taken into account.
+See BIP64 for input and output serialization (relevant for `bin` and `hex` output formats):
 https://github.com/bitcoin/bips/blob/master/bip-0064.mediawiki
```

Comment:
Optionally, could put the url into "BIP64" and replace the colon at the end of the sentence with a full stop.

FWIW, "serialisation" isn't incorrect; IIRC it's the English spelling except in North America.

Reply:
Good idea, done. Also add a motivation hint in PR description and the commit body on why "serialization" is preferred over "serialisation" (I agree that it is not incorrect from a language perspective and the reason for change is only consistency).